bin2tif/readme.md

```diff
@@ -14,6 +15,80 @@ _Output_
+
+       ii. Predict fov for each image should be:
+
+                  fix_fov = fov_in_2_meter*(camera_height/2)
+
+       iii. In implementing the stitching process, we required two magic numbers that are computed experimentally to get the correct geometric alignment.  Our experimentally determined values are:
+
+           HEIGHT_MAGIC_NUMBER = 1.64
+
+           PREDICT_MAGIC_SLOPE = 0.574
+
+           And they are used in the following way:
+
+		            predict_plant_height = PREDICT_MAGIC_SLOPE * camHeight
+
+		            camH_fix = camHeight + HEIGHT_MAGIC_NUMBER - predict_plant_height
+
+		            fix_fov_x = fov_x*(camH_fix/2)
+
+		            fix_fov_y = fov_y*(camH_fix/2)
+   ```
+   d. Compute geo-reference bounding box
+   ```
+        i. Convert coordinates from Scanalyzer to MAC coordinates using formula from [https://terraref.gitbooks.io/terraref-documentation/content/user/geospatial-information.html](https://terraref.gitbooks.io/terraref-documentation/content/user/geospatial-information.html)
+
+        ii. Use utm tools to convert coordinates from MAC to lat/lon
+   ```
+   e. Using osgeo.gdal, associate with geo-bounding box, create geotiff.
+
+### Parameters
+
+HEIGHT_MAGIC_NUMBER and PREDICT_MAGIC_SLOPE were applied when we estimate the field of view, this is a testing based number.
+
+The geo-reference bounding box is based on an assumption that image is aligned to geographic coordinates, so that moving up in the image corresponds to moving exactly north.
+
+### Limitations
+
+1. Any stitched image introduces new artifacts into the image data; it always introduces edges at the boundary of where one image turns into another --- either an explicitly black line boundary or an implicit boundary that is there because you can't exactly stitch images of a complicated 3D world (without making a full 3D model). Even if you could stitch them, the same bit of the world is usually a different brightness when viewed from different directions.
+
+2. The stitched full field image may have artifacts that arise from harsh shadows in some imaging conditions.
+
+3. One of the artifacts is duplication of area, this is unavoidable without a much more complex stitching algorithm that implicitly infers the 3D structure of the ground. The justification for not going for such a rich representation is that:
+
+    1. for the plants, since they move, it would be impossible not to have artifacts at the edges of the image, and
+
+    2. for the ground, we judged that small stitching errors were not worth the (substantial) additional effort to build the more complete model.
+
+### Failure Conditions
+
+* If the camera is moved in the gantry box, then the magic numbers may have to be recalculated or experimentally determined.
+
+* If the camera is not aligned north-south, then the geo-bounding box may not be accurate.
+
+
+## Implementation
```

Comment:
@weiqin61 and/or @ZongyangLi it would be nice to have a list of files and what they do